packages/dm-core/src/domain/types.ts

```diff
@@ -1,21 +1,22 @@
 export type TAttribute = {
-  type?: string
   name: string
-  dimensions: string
+  type: string
   attributeType: string
+  dimensions?: string
   optional?: boolean
   contained?: boolean
   description?: string
   label?: string
   enumType?: string
-  default?: string
+  default?: any
 }
 
 export type TBlueprint = {
   name: string
   type: string
-  description: string
-  attributes: TAttribute[]
-  uiRecipes?: any[]
-  storageRecipes?: any[]
+  description?: string
+  extends?: string[]
+  attributes?: TAttribute[]
+  _meta_?: any[]
```

Comment:
is _meta_ a TGenericObject?

Reply:
No. _meta_ has its own blueprint, and on of its attributes has its own blueprint... I didn't see the point in making types for this since I doubt anyone will use the meta attribute in frontend